overlord/devicestate/devicestate.go

```diff
@@ -267,3 +281,13 @@ func CanManageRefreshes(st *state.State) bool {
 
 	return false
 }
+
+func canRefreshOnMetered(st *state.State) bool {
+	tr := config.NewTransaction(st)
+	var holdOnMetered bool
+	if err := tr.GetMaybe("core", "refresh.hold-on-metered", &holdOnMetered); err != nil {
+		return true
```

Comment:
I personally would prefer that we *cannot* refresh on metered by default. But I understand this is a matter of debate.

Reply:
I'd prefer that we couldn't by default, either.

Reply:
Sounds good to me, I can rename it to eg. `refresh.allow-on-metered` and update the code with reverse logic.